lib/web_ui/lib/src/engine/canvaskit/renderer.dart

```diff
@@ -409,8 +414,53 @@ class CanvasKitRenderer implements Renderer {
   ui.ParagraphBuilder createParagraphBuilder(ui.ParagraphStyle style) =>
     CkParagraphBuilder(style);
 
+  ({
+    ui.Scene scene,
+    ui.FlutterView view,
+    Completer<void> completer
+  })? _currentRender;
+  ({
+    ui.Scene scene,
+    ui.FlutterView view,
+    Completer<void> completer
+  })? _nextRender;
```

Comment:
Can the record definition be moved into a typedef? Not only would that dedupe the definitions, but it would also allow the record structure to be documented. [Example](https://github.com/flutter/engine/blob/c9268c7db03c1d45431ee34e9e671cf98871037e/lib/web_ui/lib/src/engine/semantics/focusable.dart#L75).

Reply:
Ooo! Records

Reply:
Done.